conventions/src/main/kotlin/otel.errorprone-conventions.gradle.kts

```diff
@@ -4,82 +4,82 @@ plugins {
   id("net.ltgt.errorprone")
 }
 
-dependencies {
-  errorprone("com.google.errorprone:error_prone_core")
-}
-
-val disableErrorProne = gradle.startParameter.projectProperties.get("disableErrorProne")?.toBoolean()
-  ?: false
+val disableErrorProne = properties["disableErrorProne"]?.toString()?.toBoolean() ?: false
 
 tasks {
   withType<JavaCompile>().configureEach {
-    options.errorprone {
-      isEnabled.set(!disableErrorProne)
-
-      disableWarningsInGeneratedCode.set(true)
-      allDisabledChecksAsWarnings.set(true)
-
```

Comment:
this change is mostly whitespace, recommend clicking on github option to hide whitespace (I wish there was way to make that the default)